pyproject.toml

```diff
@@ -148,17 +148,21 @@ markers = [
 testpaths = "tests"
 
 [tool.ruff]
-ignore = [
-  "RUF012"  # mutable-class-default
-]
 line-length = 120
-select = [
-  "F",  # pyflakes
+# Ref: https://docs.astral.sh/ruff/configuration/
+src = ["fuji_server"]
+target-version = "py311"
+
+[tool.ruff.lint]
+extend-select = [
   "I",  # isort
   "UP",  # pyupgrade
   "RUF"  # ruff
 ]
-target-version = "py311"
+ignore = [
+  "E722",  # bare-except
```

Comment:
This should be enabled as well, but I couldn't figure out some of the necessary exceptions to use.

Reply:
avoiding bare excepts should be enough  I think it can stay like this

Reply:
You mean leave it as is?
There are 16 bare excepts in the code base:

https://github.com/search?q=repo%3Apangaea-data-publisher%2Ffuji%20except%3A&type=code